libs/SFMLResource/GraphicsFontResource.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** GraphicsFontComponent
+*/
+
+#ifndef GRAPHICSFONTRESOURCE_HPP_
+#define GRAPHICSFONTRESOURCE_HPP_
+
+#include "Resource/Resource.hpp"
+#include <SFML/Graphics.hpp>
+
+namespace ecs
+{
+    /// @brief This resource class stores the font in order to draw a Graphical text resource.
+    /// It justs exist for the Graphical text resource.
+    class GraphicsFontResource : public Resource {
+        public:
+            /// @brief The Graphical SFML font to be used with Graphical text resource.
+            /// @note Only used with Graphical text resource.
+            sf::Font font;
```

Comment:
Can you not specify the targeted file (.ttf or other font extension) to its creation? 
If possible, you should add the possibility to do so.

Reply:
I add a constructor with the possibility to pass a file as parameters